components/nimbus/src/evaluator.rs

```diff
@@ -29,21 +32,52 @@ impl Bucket {
     }
 }
 
+#[cfg(feature = "stateful")]
 #[derive(Serialize, Deserialize, Debug, Clone, Default)]
 pub struct TargetingAttributes {
     #[serde(flatten)]
     pub app_context: AppContext,
     pub language: Option<String>,
     pub region: Option<String>,
     pub is_already_enrolled: bool,
-    #[cfg(feature = "stateful")]
     pub days_since_install: Option<i32>,
-    #[cfg(feature = "stateful")]
     pub days_since_update: Option<i32>,
-    #[cfg(feature = "stateful")]
     pub active_experiments: HashSet<String>,
 }
 
+#[cfg(not(feature = "stateful"))]
+#[derive(Serialize, Deserialize, Debug, Clone, Default)]
+pub struct TargetingAttributes {
+    #[serde(flatten)]
+    pub app_context: AppContext,
+    #[serde(flatten)]
+    pub request_context: RequestContext,
+    pub language: Option<String>,
+    pub region: Option<String>,
+    pub is_already_enrolled: bool,
```

Comment:
Nit: I don't think makes sense in this context.

Reply:
No feature-gate required! It works without it fine